test/api_tester.py

```diff
@@ -0,0 +1,257 @@
+            ApiTester.imaging_api.delete_folder(requests.DeleteFolderRequest(ApiTester.temp_folder, ApiTester.test_storage, True))
+
+    def __create_api_instance(self, app_key=None, app_sid=None, base_url=None,
+                              api_version=None, debug=False):
+        if not app_key:
+            app_key = ApiTester._app_key
+
+        if not app_sid:
+            app_sid = ApiTester._app_sid
+
+        if not base_url:
+            base_url = ApiTester._base_url
+
+        if not api_version:
+            api_version = ApiTester._api_version
+
+        if app_key == ApiTester._app_key or app_sid == ApiTester._app_sid:
+            print("Access data isn't set explicitly. Trying to obtain it from environment variables.")
+
+            app_key = os.environ.get('AppKey')
+            app_sid = os.environ.get('AppSid')
+            base_url = os.environ.get('ApiEndpoint')
+            api_version = os.environ.get('ApiVersion')
+
+        if not (app_key and app_sid and base_url and api_version):
+            print(
+                "Access data isn't set completely by environment variables. Filling unset data with default values.")
+
+        if not api_version:
+            api_version = ApiTester._api_version
+            print('Set default API version')
+
+        server_access_info = os.path.join(ApiTester._local_test_folder, ApiTester._server_access_file)
+        with open(server_access_info) as f:
+            server_file_info = json.load(f)
+
+        if server_file_info:
+            if not app_key:
+                app_key = server_file_info['AppKey']
+                print('Set default App key')
+
+            if not app_sid:
+                app_sid = server_file_info['AppSid']
+                print('Set default App SID')
+
+            if not base_url:
+                base_url = server_file_info['BaseURL']
+                print('Set default Base URL')
+
+        if not (app_key and app_sid and base_url and api_version):
+            raise ValueError('Please, specify valid access data (AppKey, AppSid, Base URL)')
+
+        print('App key: ' + app_key)
+        print('App SID: ' + app_sid)
+        print('Storage: ' + ApiTester.test_storage)
+        print('Base URL: ' + base_url)
+        print('API version: ' + api_version)
+
+        api_client = ApiClient()
+        api_client.configuration.host = base_url
+        api_client.configuration.api_key['api_key'] = app_key
+        api_client.configuration.api_key['app_sid'] = app_sid
+        api_client.configuration.api_version = api_version
+
+        ApiTester.imaging_api = ImagingApi(api_client)
+
+        ApiTester.input_test_files = ApiTester.imaging_api.get_files_list(
+            requests.GetFilesListRequest(ApiTester.original_data_folder, ApiTester.test_storage)).value
+
+    def __request_tester(self, test_method_name, save_result_to_storage, parameters_line, input_file_name,
```

Comment:
@s-zubov Please, use the _test_method_name_ parameter to print it to console or show the place where it's printed.

Reply:
I've printed a case name in a test setup. Changed it to print `test_method_name` in request_tester